### PR TITLE
update spanish translations

### PR DESCRIPTION
**Pull request by Lane**
Adds in updates for
  - max locations error
  - census demographics label
  - ranking panel robotext

### CODE REVIEW

File: src/assets/i18n/es.json [hunk 1]
```diff
@@ -63,7 +63,7 @@
     "SEND_DATA": "Enviar datos",
     "AUTOCOMPLETE_HINT": "Cuando esté disponible, use las flechas hacia arriba y hacia abajo para revisar e ingresar para seleccionar.",
     "LOADING": "Cargando",
-    "TRACT_SINGULAR": "Sección Censal",
+    "TRACT_SINGULAR": "Sección Censal",
```
Comment (pjsier) [inline src/assets/i18n/es.json]: It looks like this is just a whitespace addition?

Comment (Lane) [reply src/assets/i18n/es.json]: whoops, will drop

**Comment by pjsier** [comment]
It looks like the Census Demographics translation goes right up against the borders of the table on mobile. Not sure how we want to handle that, but everything else looks good

screen shot 2018-04-13 at 1 59 12 pm

**Comment by Lane** [comment]
image

Added some padding so it wraps to a second line.

**Pull request merged**